pages/photon.vue

```diff
@@ -1,19 +1,222 @@
 <template>
-  <div>Content goes here</div>
+  <div>
+    <header>Header</header>
+    <section class="photon__section photon__section_light">
+      <div class="photon__container">
+        <h2 class="copy__title">
+          Lorem ipsum
+        </h2>
+        <div class="bx--row">
+          <div class="bx--col-lg-6 section-mobile-spacing">
```

Comment:
Could you tried with `bx--col-md-4` and give me your opinion. I think it could look better for medium sizes and I would like to know if you agree :)

Reply:
I like that 👍